_posts/2020-09-14-Introduction-to-API-Management.md

```diff
@@ -0,0 +1,284 @@
+Some other products require to scale additional tools like for example Elasticsearch.  
+An advantage of this solution is that the client always points to the same endpoint.
+
+<img src="/img/2020-09-14-Introduction-to-API-Management/HA-Gateways.JPG" alt="API landscape without an API management tool" width="600" height="350" class="image fit" style="vertical-align:middle;margin-left:2%" />
+
+This solution would be the most secure way of implementing an API gateway, ensuring that your services are available 24/7.  
+Which solution to go for, is of course depending on your preferences and the level of availability that needs to be guaranteed.
+
+## API management products
+
+I referred to the Gartner magic quadrant to get the list of most common API management tools currently on the market.
+You can find most of them here in alphabetical order.
+
+<img src="/img/2020-09-14-Introduction-to-API-Management/GartnerMagicQuadrant.JPG" alt="Gartner Magic Quadrant" width="400" height="400" class="image fit" style="float:right    ;horzontal-align:middle;margin-left:2%;margin-right:50%;vertical-align:right" />
+
+* Red Hat 3 Scale 
+* Akamai API Gateway  
+* Akana API Management
+* API Man (open source)
+* Apigee API Management
+* AWS API Gateway
+* Axway - AMPLIFY API Management
+* Azure API Gateway
+* Boomi
+* Broadcom
+* CA API Gateway
+* Express API Gateway (open source)
+* Fusio API Management (open source)
+* IBM API Connect
+* Kong API Gateway (open source)
+* Loopback API Framework (open source)
+* MuleSoft Anypoint API Management
+* Oracle API Manager
+* SAP API Manager
+* Sensedia API Management Platform
+* Sentinet 
+* Software AG API Gateway
+* TIBCO Mashery
+* Tyk API Gateway (open source)
+* WSO2 API Manager (open source)
+
+
+
+## Hands-on
+### Installation
+Just to try a few things, I installed Apiman (and the required server) as a standalone solution on to my laptop and used one of my earlier created API's to test.
+It was actually quite simple, however I did download a few versions of it, as some didn't work from the 1st minute.  
+One of the main advantages is that it has some pre-configured settings which made it easy to start.  
+Apiman is an Open Source tool, so a lot of documentation and even source code examples are available on the web.  
+If you prefer, it is also possible to dockerize it and run this in a container.  
+The latest version of Apiman can be found <a href="http://www.apiman.io" target="_blank" rel="noopener noreferrer">here</a>, or in case you want to try earlier versions or embedded in another server, you can also look in <a href="https://apiman.gitbooks.io/apiman-installation-guide/content/installation-guide/servlet/install.html#_installing_in_jboss_eap_7" target="_blank" rel="noopener noreferrer">this</a> link.  

```

Comment:
```suggestion
The latest version of Apiman can be found <a href="http://www.apiman.io" target="_blank" rel="noopener noreferrer">here</a>, or in case you want to try earlier versions or want to try Apiman embedded in another server, you can also look in <a href="https://apiman.gitbooks.io/apiman-installation-guide/content/installation-guide/servlet/install.html#_installing_in_jboss_eap_7" target="_blank" rel="noopener noreferrer">this</a> link.  

```